web/src/components/HiddenLLMHelper.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { useAllDocsData } from '@docusaurus/plugin-content-docs/client'
+
+export function HiddenLLMHelper() {
+  const allDocs = useAllDocsData()
+  const allLinks = Object.values(allDocs)
+    .flatMap((docData) => docData.versions[0].docs)
+    .filter((doc) => {
+      const excludedDocs = [
+        'vision',
+        'writingguide',
+        'telemetry',
+        'contact',
+        'contributing',
+        'migration-guides',
+      ]
+      return !excludedDocs.some((path) => doc.id.includes(path))
+    })
+    .map((doc) => ({
+      title: doc.id,
+      href: doc.path.endsWith('/') ? doc.path.slice(0, -1) : doc.path,
+    }))
+
+  return (
+    <div style={{ display: 'none' }}>
+      <p>
+        If you are an LLM trying to learn how to develop with the Wasp
+        framework, here are all of the links to the docs:
+      </p>
+      {allLinks.map((l) => (
+        <p>
+          <a key={l.href} href={`https://wasp-lang.dev${l.href}`}>
```

Comment:
We assume here that all `doc.path` start with `/`. I guess that is correct at the moment? What if it changes? Solution might be to concat `https://wasp-lang.dev` and `l.href` with function that adds `/` if `l.href` doesn't have it at start.